src/auth/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const jwtConstants = {
+  secret:
+    'DO NOT USE THIS VALUE. INSTEAD, CREATE A COMPLEX SECRET AND KEEP IT SAFE OUTSIDE OF THE SOURCE CODE.',
```

Comment:
**issue**: Storing secrets in source code is insecure. Use environment variables instead.